### PR TITLE
Added documentation examples install_yamls environment and default vars.

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,6 +8,35 @@ Run the following command to get your new role ready:
...
+    name: 'install_yamls_makes'
+    tasks_from: 'make_edpm_deploy'
+```
+
+```
+# Fetch openstackdataplane resources from the default namespace declared in install_yamls
+# Makefile. If the NAMESPACE has been overidden by cifmw_install_yamls_vars this variable
+# already points to the overriden value.
+- name: Get info about dataplane node
+    environment:
```
Comment (cjeanner) [inline CONTRIBUTING.md:34]:
```suggestion
  environment:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,6 +8,35 @@ Run the following command to get your new role ready:
...
+    PATH: "{{ cifmw_path }}"
+    ansible.builtin.command: |
```
Comment (cjeanner) [inline CONTRIBUTING.md:36]:
```suggestion
  ansible.builtin.command: |
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -8,6 +8,35 @@ Run the following command to get your new role ready:
...
 
+### Consuming install_yamls varibles
+CI Framework sets a couple of facts that are useful enough to be ment, the
+`cifmw_install_yamls_vars`, that contains all the install_yamls variables
+that should be passed to any instal_yaml target, and the `cifmw_install_yamls_default`,
+that contains the default values of every possible parameter of the install_yamls
+Makefiles.
+
+Here is an example, based on a common use-case, on how to use those variables
+```
```
Comment (cjeanner) [inline CONTRIBUTING.md:19]:
```suggestion
```YAML
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -8,6 +8,35 @@ Run the following command to get your new role ready:
...
+- name: Deploy EDPM
+  vars:
+    make_edpm_deploy_env: "{{ cifmw_install_yamls_environment | combine({'PATH': cifmw_path }) }}"
+    make_edpm_deploy_dryrun: "{{ cifmw_edpm_deploy_dryrun | bool }}"
+  ansible.builtin.include_role:
+    name: 'install_yamls_makes'
+    tasks_from: 'make_edpm_deploy'
+```
+
+```
```
Comment (cjeanner) [inline CONTRIBUTING.md:29]:
```suggestion
```YAML
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -8,6 +8,35 @@ Run the following command to get your new role ready:
...
+that contains the default values of every possible parameter of the install_yamls
+Makefiles.
+
+Here is an example, based on a common use-case, on how to use those variables
+```
+- name: Deploy EDPM
+  vars:
+    make_edpm_deploy_env: "{{ cifmw_install_yamls_environment | combine({'PATH': cifmw_path }) }}"
+    make_edpm_deploy_dryrun: "{{ cifmw_edpm_deploy_dryrun | bool }}"
+  ansible.builtin.include_role:
```
Comment (cjeanner) [inline CONTRIBUTING.md:24]:
```suggestion
  ansible.builtin.import_role:
```

**Review by raukadah — APPROVED** [review]
/lgtm
/approve

**Pull request merged**